qemu/tests/win_heavyload.py

```diff
@@ -130,7 +130,7 @@ def add_option(cmd, key, val):
 
         error.context("Verify vm is alive", logging.info)
         utils_misc.wait_for(vm.verify_alive,
-                            timeout=test_timeout, step=steping)
+                            timeout=test_timeout*60, step=steping)
```

Comment:
heavyload.exe use this value in minutes,  and utils_miss.wait_for use this value in seconds.